spec/models/glossary_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Glossary, type: :mode do
+  let(:glossaryMock) { build(:glossary) }
+  let(:glossaryMock2) { build(:glossary) }
+
+  context 'creation' do
+    it 'will mark an empty glossary instance as invalid' do
+      glossary = Glossary.new
+      glossary.valid?
+      expect(glossary.errors.details[:term]).to include(
+        a_hash_including(error: :blank)
+      )
+    end
+
+    it 'will mark entries without a term as invalid' do
+      glossary = Glossary.new(definition: glossaryMock.definition)
+      glossary.valid?
+      expect(glossary.errors.details[:term]).to include(
+        a_hash_including(error: :blank)
+      )
+    end
+
+    it 'will mark entries without a definition as invalid' do
+      glossary = Glossary.new(term: glossaryMock.term)
+      glossary.valid?
+      expect(glossary.errors.details[:definition]).to include(
+        a_hash_including(error: :blank)
+      )
+    end
+
+    it 'adds valid entries to the DB' do
+      glossary = Glossary.new(
+        term: glossaryMock.term,
+        definition: glossaryMock.definition
+      )
+      expect { glossary.save }.to change(Glossary, :count).by(1)
+    end
+  end
+
+  context 'modification' do
+    before(:each) do
+      @glossary = Glossary.create(
```

Comment:
You've changed syntax and are now using instance variables - why did you move away from `let` syntax on top of `FactoryBot.create`?